web/modules/custom/da_poll/src/Controller/PollController.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace Drupal\da_poll\Controller;
+
+use Drupal\Component\Utility\Xss;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Database\Database;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Url;
+use Drupal\da_poll\Entity\PollInterface;
+
+/**
+ * Class PollController.
+ *
+ *  Returns responses for Poll routes.
+ *
+ * @package Drupal\da_poll\Controller
+ */
+class PollController extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * Displays a Poll  revision.
+   *
+   * @param int $da_poll_revision
+   *   The Poll  revision ID.
+   *
+   * @return array
+   *   An array suitable for drupal_render().
+   */
+  public function revisionShow($da_poll_revision) {
+    $da_poll = $this->entityManager()
```

Comment:
The entity manager service is deprecated. Please inject the entity type manager and use that.